src/main/java/com/desiato/puresynth/dtos/ErrorLoginResponseDTO.java

```diff
@@ -1,25 +1,18 @@
 package com.desiato.puresynth.dtos;
 
-public class ErrorLoginResponseDTO implements LoginResponseDTO {
-    private final String message;
-    private final boolean success = false;
-
-    public ErrorLoginResponseDTO(String message) {
-        this.message = message;
-    }
-
+public record ErrorLoginResponseDTO(String message, boolean success) implements LoginResponseDTO {
```

Comment:
You don't need `success` field. It's enough to just override the method.

Reply:
removed